simapp/ante_handler.go

```diff
@@ -0,0 +1,35 @@
+package simapp
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/x/auth/ante"
+	"github.com/cosmos/cosmos-sdk/x/auth/signing"
+	"github.com/cosmos/cosmos-sdk/x/gov/types"
+	channelkeeper "github.com/cosmos/cosmos-sdk/x/ibc/core/04-channel/keeper"
+	ibcante "github.com/cosmos/cosmos-sdk/x/ibc/core/ante"
+)
+
+func NewAnteHandler(
+	ak ante.AccountKeeper, bankKeeper types.BankKeeper,
+	sigGasConsumer ante.SignatureVerificationGasConsumer,
+	signModeHandler signing.SignModeHandler,
+	IBCChannelkeeper channelkeeper.Keeper,
```

Comment:
```suggestion
	channelKeeper channelkeeper.Keeper,
```

is in keeping with naming style we already have